flocker/common/version.py

```diff
@@ -15,11 +15,11 @@
     # Weekly release
     r"(dev(?P<weekly_release>[0-9]+))?"
     # The documentation release
-    r"(\+doc(?P<documentation_revision>[0-9]+))?"
+    r"(\.post(?P<documentation_revision>[0-9]+))?"
     # Development version
-    r"(-(?P<commit_count>[0-9]+)-g(?P<commit_hash>[0-9a-f]+))?"
+    r"(\+(?P<commit_count>[0-9]+).g(?P<commit_hash>[0-9a-f]+))?"
```

Comment:
This is exactly the point of this change. The old versioning scheme of versioneer isn't PEP 440 compatible. The new one is.